samples/v2/instance-config-get-operations.js

```diff
@@ -0,0 +1,78 @@
+/**
+ * Copyright 2024 Google LLC
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+// sample-metadata:
+//  title: Lists the instance configuration operations.
+//  usage: node instance-config-get-operations <PROJECT_ID>
+
+'use strict';
+
+function main(projectId = 'my-project-id') {
+  // [START spanner_list_instance_config_operations]
+
+  /**
+   * TODO(developer): Uncomment the following lines before running the sample.
+   */
+  // const projectId = 'my-project-id';
+
+  // Imports the Google Cloud client library
+  const {protos} = require('@google-cloud/spanner');
+  const {InstanceAdminClient} = require('@google-cloud/spanner/build/src/v1');
+
+  // creates an instance admin client
+  const instanceAdminClient = new InstanceAdminClient({
+    projectId: projectId,
+  });
+  async function getInstanceConfigOperations() {
+    // Lists the instance config operations.
+    try {
+      console.log(
+        `Getting list of instance config operations on project ${projectId}...\n`
+      );
+      const [instanceConfigOperations] =
+        await instanceAdminClient.listInstanceConfigOperations({
+          parent: instanceAdminClient.projectPath(projectId),
+          filter:
+            '(metadata.@type=type.googleapis.com/google.spanner.admin.instance.v1.CreateInstanceConfigMetadata)',
```

Comment:
We should add a comment here that explains that this filters for only `InstanceConfig` operations that create a new instance config.